### PR TITLE
Session: nette-browser cookie may be sent with secure flag

### PR DESCRIPTION
**Pull request by PavelJurasek**

### CODE REVIEW

File: src/Http/Session.php [hunk 1]
```diff
@@ -545,7 +545,7 @@ private function sendCookie()
 		);
 		$this->response->setCookie(
 			'nette-browser', $_SESSION['__NF']['B'],
-			Response::BROWSER, $cookie['path'], $cookie['domain']
+			Response::BROWSER, $cookie['path'], $cookie['domain'], $cookie['secure']
```
Comment (JanTvrdik) [inline src/Http/Session.php]: * Why not pass httponly option as well?
* This has been removed in https://github.com/nette/http/commit/0aed1a2bb65f42cb8a32f3909fd593538ac52f57, not sure why

Comment (PavelJurasek) [inline src/Http/Session.php]: You are right, I'll add httponly too.

**Comment by dg** [comment]
What's the reason?

**Comment by PavelJurasek** [comment]
Mixed content is being sent when site in on HTTPS.

**Comment by dg** [comment]
It is error message? In which browser?

**Comment by PavelJurasek** [comment]
There is no error nor warning in browsers. At least not when HSTS is used. 
Mozilla Observatory: 
screen

**Comment by dg** [comment]
Closed by #103

**Pull request closed without merging**